.eslintrc.js

```diff
@@ -55,6 +53,12 @@ module.exports = {
       files: [ '**/*.ts' ],
 
       parser: '@typescript-eslint/parser',
+      plugins: ['@typescript-eslint'],
+      extends: [
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:import/typescript',
+        'prettier/@typescript-eslint'
```

Comment:
`prettier/@typescript-eslint` turns off the `@typescript-eslint` rules that conflict with `prettier`